qe-tests/cypress/integration/tests/vmware/config_forexternal_cluster.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  LoginData,
+  MappingData,
+  PlanData,
+  TestData,
+  OcpVirtData,
+  VmwareProviderData,
+  MappingPeer,
+} from '../../types/types';
+import { providerType, storageType } from '../../types/constants';
+
+const url = Cypress.env('url');
+const user_login = 'kubeadmin';
+const user_password = Cypress.env('pass');
+
+export const loginData: LoginData = {
+  username: user_login,
+  password: user_password,
+  url: url,
+};
+
+export const target_ocpData: OcpVirtData = {
+  type: 'OpenShift Virtualization',
+  name: 'mgn01',
+  url: 'https://api.mgn01.cnv-qe.rhcloud.com:6443',
+  //saToken: 'sha256~i3ZkqZC5P41Vs3SWSyL7Z1hnF4vifp-4CTKqsasXW6s',
```

Comment:
Please avoid committing with comments like this

Reply:
TBH, this info should be moved to a private repo .